eventsource-client/README.md

```diff
@@ -0,0 +1,46 @@
+# eventsource-client
+
+Client for the [Server-Sent Events] protocol (aka [EventSource]).
+
+[Server-Sent Events]: https://html.spec.whatwg.org/multipage/server-sent-events.html
+[EventSource]: https://developer.mozilla.org/en-US/docs/Web/API/EventSource
+
+## Requirements
+
+Requires tokio.
+
+## Usage
+
+Example that just prints the type of each event received:
+
+```rust
+use eventsource_client::Client;
+
+let mut client = Client::for_url("https://example.com/stream")?
+    .header("Authorization", "Basic username:password")?
+    .build();
+
+client.stream()
+    .for_each(|event| {
+      Ok(println!("got an event: {}", event.event_type))
+    })
+    .map_err(|e| println!("error streaming events: {:?}", e));
+```
+
+(Some boilerplate omitted for clarity; see [examples directory] for complete,
+working code.)
+
+[examples directory]: ../../tree/master/examples
+
```

Comment:
I suspect we can't have this link work properly. See also broken examples link in https://github.com/launchdarkly/rust-server-sdk-private

Reply:
We can if we just make it the full path right?